<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioControlWindow</class>
 <widget class="QWidget" name="AudioControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>183</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioControlWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wgt_main{
background:rgba(38,38,38,1);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#button_MIC{
background:transparent;
border-image: url(:/TQLive/images/live tool_icon_mike_normal.png);
}
#button_MIC:hover{
	border-image: url(:/TQLive/images/live tool_icon_mike_hover.png);
}
#button_MIC:checked{
border-image: url(:/TQLive/images/live tool_icon_mike no_normal.png);
}
#button_MIC:checked:hover{
border-image: url(:/TQLive/images/live tool_icon_mike no_hover .png);
}
#button_JBL{
background:transparent;
	border-image: url(:/TQLive/images/live tool_icon_the horn_normal.png);
}
#button_JBL:hover{
border-image: url(:/TQLive/images/live tool_icon_the horn_hover.png);
}
#button_JBL:checked{
border-image: url(:/TQLive/images/live tool_icon_mute_normal.png);
}
#button_JBL:checked:hover{
border-image: url(:/TQLive/images/live tool_icon_mute_hover.png);
}
QSlider{
background:transparent;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>2</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>2</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:1px solid #585858;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_MIC">
            <property name="minimumSize">
             <size>
              <width>13</width>
              <height>12</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>13</width>
              <height>12</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_MIC">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_JBL">
            <property name="minimumSize">
             <size>
              <width>13</width>
              <height>12</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>13</width>
              <height>12</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_JBL">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
